### ISSUE TITLE
3.2 Functions -  python code example gives TypeError

### ISSUE THREAD

**Issue by Saveyy** [picoCTF/ctf-primer#27]
The Code in the Primer gives TypeError.
```
Input a number:
10
Traceback (most recent call last):
  File "HelloWorld.py", line 8, in <module>
    if even_odd(my_number):
  File "HelloWorld.py", line 2, in even_odd
    if (x % 2 == 0):
TypeError: not all arguments converted during string formatting
```
Added int to input variable of the function fixed it for me. 

```
def even_odd(input):
        if (int(input) % 2 == 0):
                return True
        else:
                return False


print("input number")
mynumber = input()
if (even_odd(mynumber)):
        print("it's even!")
else:
        print("it's odd")
```

**Comment by Saveyy** [comment]
I got this using the webpage-cli

**Comment by Reckless630** [comment]
changed it up a little bit an it works like this:

my_number = int(input("Enter a number:"))
def even_odd(my_number):
    if(my_number % 2 == 0):
        print(my_number," is an even")
    else:
        print(my_number, " is an odd")
even_odd(my_number)

**Comment by Reckless630** [comment]
i solved the problem like that

```
my_number = int(input("Enter a number:"))
def even_odd(my_number):
    if(my_number % 2 == 0):
        print(my_number," is an even")
    else:
        print(my_number, " is an odd")
even_odd(my_number)
```

**Comment by hiliang-cmu** [comment]
Resolved w/ e37bb49
